dll/ntdll/ldr/ldrinit.c

```diff
@@ -1479,15 +1485,15 @@ LdrpInitializeExecutionOptions(PUNICODE_STRING ImagePathName, PPEB Peb, PHANDLE
                                    sizeof(RtlpShutdownProcessFlags),
                                    NULL);
 
-        LdrQueryImageFileKeyOption(KeyHandle,
-                                   L"MinimumStackCommitInBytes",
-                                   REG_DWORD,
-                                   &MinimumStackCommit,
-                                   sizeof(MinimumStackCommit),
-                                   NULL);
+        Status = LdrQueryImageFileKeyOption(KeyHandle,
+                                            L"MinimumStackCommitInBytes",
+                                            REG_DWORD,
+                                            &MinimumStackCommit,
+                                            sizeof(MinimumStackCommit),
+                                            NULL);
 
         /* Update PEB's minimum stack commit if it's lower */
-        if (Peb->MinimumStackCommit < MinimumStackCommit)
+        if (NT_SUCCESS(Status) && Peb->MinimumStackCommit < MinimumStackCommit)
```

Comment:
```suggestion
        if (NT_SUCCESS(Status) && (Peb->MinimumStackCommit < MinimumStackCommit))
```

Reply:
Why? I would rather not.

Reply:
This is part of our coding style. All sub-expressions need to be wrapped in braces for clarity.

Reply:
> This is part of our coding style. All sub-expressions need to be wrapped in braces for clarity.

Is it? I see nothing about that on https://reactos.org/wiki/Coding_Style .
I would argue about unneeded parentheses being clearer or confusing.